doc/version_1/040_treesForVulMgmt.md

```diff
@@ -22,6 +22,11 @@ Relatedly, C-level executives and public policy professionals often make, shape,
 
 Stakeholders with different responsibilities in vulnerability management have largely different decisions to make. This section focuses on the differences among organizations based on their vulnerability management responsibilities. Some decision makers may have different responsibilities in relation to different software. For example, an Android app developer is a developer of the app, but is a deployer for any changes to the Android OS API. This situation is true for libraries in general. A web browser developer makes decisions about applying patches to DNS lookup libraries and transport layer security (TLS) libraries. A video game developer makes decisions about applying patches released to the Unreal Engine. A medical device developer makes decisions about applying patches to the Linux kernel. The list goes on.  Alternatively, one might view applying patches as, de facto, including some development and distribution of the updated product. Or one might take the converse view, that development, de facto, includes updating libraries. Either way, in each of these examples (mobile device apps, web browsers, video games, medical devices), we recommend that the professionals making genuine decisions do three things: (1) identify the decisions explicitly, (2) describe how they view their role(s), and (3) identify which software projects their decision relates to. If their decisions are explicit, then the decision makers can use the recommendations from this document that are relevant to them.
 
+<<new subtitle here?>>
```

Comment:
How about "Vulnerability Management Actions"?

Reply:
I plan to add more details under this section to answer #79 and make the links to the decisions that are influenced by remediation. 
What about "Enumerating Vulnerability Management Actions"?

Reply:
Works for me.